templates/templates/_navigation-enterprise-h.html

```diff
@@ -92,10 +93,10 @@ <h4 class="p-heading-four is-dense">
             </a>
           </div>
           <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
-            <a href="/support">Enterprise support&nbsp;&rsaquo;</a>
+            <a href="/support">Enterprise Support&nbsp;&rsaquo;</a>
           </h4>
           <ul class="p-text-list--small is-bordered">
-            <li class="p-list__item u-show--small u-hide--medium u-hide--large"><a href="/support">Get support</a></li>
+            <li class="p-list__item"><a href="/support">Get support</a></li>
```

Comment:
plz add
```
li class="p-list__item u-show--small u-hide--medium u-hide--large"><a href="/support">Get support</a></li>
```

Reply:
thanks, I shouldn't have done all this before a holiday!

Reply:
You shouldn't be doing it on holiday!!

Naughty Peter. But thank you nonetheless

😄